Source/items.cpp

```diff
@@ -1127,8 +1127,8 @@ void SaveItemAffix(ItemStruct &item, const PLStruct &affix)
 
 	if (!gbIsHellfire) {
 		if (power.type == IPL_TARGAC) {
-			power.param1 = 2 << power.param1;
-			power.param2 = 6 << power.param2;
+			power.param1 = 2 << (power.param1 - 1);
+			power.param2 = 6 << (power.param2 - 1);

```

Comment:
```suggestion
			power.param1 = 1 << power.param1; // 2**param1
			power.param2 = 3 << power.param2; // 2**param2 + 2**(param2+1)
```
Halving the lhs has the same effect, saves needing to subtract as a separate operation. The comments are to try explain the operation the shifts are accomplishing.

Reply:
good catch :D